d3a_interface/dataclasses.py

```diff
@@ -1,16 +1,36 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
 from dataclasses import dataclass, field
-from typing import Dict, List
+from typing import List, Dict, Optional
 
 
 @dataclass
 class BidOfferMatch:
+    """Representation of a market match."""
     market_id: str
     bids: List[Dict]
     selected_energy: float
     offers: List[Dict]
     trade_rate: float
 
-    def serializable_dict(self):
+    def serializable_dict(self) -> Dict:
```

Comment:
Nit: since we have `from_dict()`, I think this method might be called `to_dict()`.

Reply:
I agree, however I kept the naming because we have it in all our data classes/classes (Offer, Bid, Trade, TradeInfo ....)